Source/WebCore/platform/image-decoders/jpegxl/JxlDecoderPtr.h

```diff
@@ -28,7 +28,12 @@
 #if USE(JPEGXL)
 
 #include <algorithm>
+
+#if PLATFORM(COCOA)
+#include <AppleJPEGXL/decode.h>
```

Comment:
Is this the right way to include it, not the usual `<AppleJPEGXL/AppleJPEGXL.h>?

Reply:
Weirdly, there is no AppleJPEGXL.h file. :/